pkg/rsnotify/listeners/local/go.mod

```diff
@@ -5,12 +5,10 @@ go 1.17
 require (
 	github.com/fortytw2/leaktest v1.3.0
 	github.com/google/uuid v1.1.2
-	github.com/rstudio/platform-lib/pkg/rsnotify v1.2.0
+	github.com/rstudio/platform-lib/pkg/rsnotify v1.4.0
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f
 )
 
-replace github.com/rstudio/platform-lib/pkg/rsnotify => ../../
```

Comment:
Here, and in the other listener implementations, I removed the `replace` directives. This is better, I think, since it will help us avoid false positive builds. Builds will now fail if we fail to update a listener to use the correct `rsnotify` dependency version.